src/main/resources/assets/admin/common/js/ui/button/MenuButton.ts

```diff
@@ -63,6 +65,20 @@ export class MenuButton
         this.menu.removeSeparator();
     }
 
+    toggleMenu(expanded?: boolean) {
+        if (expanded || !this.menu.hasClass('expanded')) {
+            this.expandMenu();
+        } else {
+            this.collapseMenu();
+        }
+    }
+
+    expandMenu(): void {
+        this.menu.toggleClass('expanded');
+        this.dropdownHandle.toggleClass('down');
+        this.dropdownHandle.giveFocus();
+    }
```

Comment:
The method is called "`expandMenu`", but its code actually toggles the menu.

`expandMenu` and `collapseMenu` should have similar code with reverse logic. Either `expandMenu` should use `addClass` (like `collapseMenu` is using `removeClass` now), or both of them should use `toggleClass`, but with different second argument.